libs/api-interfaces/src/lib/aggregate-views/view-models/aggregate-type.enum.ts

```diff
@@ -9,6 +9,7 @@ export const AggregateType = {
     category: 'category',
     user: 'user',
     userGroup: 'userGroup',
+    contributor: 'contributor',
```

Comment:
Great! Let's see if this breaks the world. Eventually, we want to phase out the lookup tables \ global coupling to avoid issues, but for now, I think I probably force you to implement everything up front.